packages/search/README.md

```diff
@@ -1,88 +1,159 @@
-## Description
+## Overview
 
-The `<sp-search>` element delivers a single input field with a "reset" button as well as a mangifying glass icon with which to power search interactions.
+The `<sp-search>` element is used for searching and filtering items.
+
+[View the design documentation for this component.](https://spectrum.adobe.com/page/search-field/)
 
 ### Usage
 
 [![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/search?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/search)
 [![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/search?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/search)
 [![Try it on Stackblitz](https://img.shields.io/badge/Try%20it%20on-Stackblitz-blue?style=for-the-badge)](https://stackblitz.com/edit/vitejs-vite-xnfnkgvf)
 
-```
+```zsh
 yarn add @spectrum-web-components/search
 ```
 
 Import the side effectful registration of `<sp-search>` via:
 
-```
+```js
 import '@spectrum-web-components/search/sp-search.js';
 ```
 
 When looking to leverage the `Search` base class as a type and/or for extension purposes, do so via:
 
-```
+```js
 import { Search } from '@spectrum-web-components/search';
 ```
 
-## Sizes
+### Anatomy
+
+The search field consists of several key parts:
+
+- A label/search term
```

Comment:
nit: should this just be "Label" or "Label/placeholder"? search term isnt really a part but an value from an action. could be overthinking it

Also can we remove "A" and "An" since its a bulleted list?